packages/gatsby-theme-docs/src/pages/release-notes.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
```

Comment:
<img width="268" alt="image" src="https://user-images.githubusercontent.com/1110551/78011074-6a33bc80-7343-11ea-9b54-06454ca5da83.png">

This should be a `template`, not a page

Reply:
I thought about that, and this appears to be a better/simpler approach than using a template.

Reply:
Why, we already discussed this. It should work the same was as for the content pages. Why does it need to be different?

Reply:
Using template now.